### ISSUE TITLE
Firefox crash

### ISSUE THREAD

**Issue by qazip** [netblue30/firejail#1331]
Hello,

Running firefox with firejail (firejail firefox) makes it crash:

```
firejail firefox     
Reading profile /etc/firejail/firefox.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/whitelist-common.inc
Parent pid 10326, child pid 10327
Blacklist violations are logged to syslog
Child process initialized in 78.99 ms
Warning: cannot switch euid to root
Warning: cannot switch egid to root
Warning: cannot switch euid to root
Warning: cannot switch egid to root
Warning: an existing sandbox was detected. /usr/bin/firefox will run without any additional sandboxing features
[Parent 7] ###!!! ABORT: X_ShmPutImage: BadValue (integer parameter out of range for operation); 3 requests ago: file /build/firefox-IKSm1A/firefox-53.0.3+build1/toolkit/xre/nsX11ErrorHandler.cpp, line 147
[Parent 7] ###!!! ABORT: X_ShmPutImage: BadValue (integer parameter out of range for operation); 3 requests ago: file /build/firefox-IKSm1A/firefox-53.0.3+build1/toolkit/xre/nsX11ErrorHandler.cpp, line 147
ExceptionHandler::GenerateDump cloned child 71
ExceptionHandler::SendContinueSignalToChild sent continue signal to child
ExceptionHandler::WaitForContinueSignal waiting for continue signal...

(crashreporter:72): Gdk-WARNING **: crashreporter: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.0.


Parent is shutting down, bye...

```

Firejail version:

`firejail version 0.9.46
`

**Comment by netblue30** [comment]
Are you using Arch Linux?

This is a problem with newer versions of GTK3 library. We have a fix on mainline, or you can still use 0.9.46 and modify some of your profile files in /etc/firejail directory. As root user, open /etc/firejail/firefox.profile file in a text editor and comment out ipc-namespace line (add a #, like this: https://github.com/netblue30/firejail/commit/3d828459a18716b54fd91ddc36d38134e79aa81e).

There are some other programs with the same problem. The fix is the same for all of them: comment out ipc-namespace in the profile file. I'll have a new release out shortly.

**Comment by qazip** [comment]
No, I'm using Xubuntu 16.04. Commenting ipc-namespace fixed my issue. Waiting for the new release, thank you.
